R/run.R

```diff
@@ -13,22 +16,25 @@
 #' show_in_console = FALSE)
 #' @template verbose
 #'
-#' @return Returns table showing which directories had model run and which
+#' @return Returns a table showing which directories had models run and which
 #' had errors like missing executable or Report.sso already present.
 #' The three possible messages are "model run failed", "ran model", and
 #' "unknown run status".
 #'
-#' @description First searches path for the specified executable name
-#' and returns the location of that file if found. If not found in path
-#' it then searches the specified directory. Linux systems have an
+#' @description The `run()` function checks for the executable via
+#' [r4ss::check_exe()]. This involves first checking first checking the
+#' specified 
+#' directory `dir` for the specified SS3 executable name. If it is not
+#' found in the specified
+#' directory, then it checks the PATH. Linux systems may have an
 #' existing executable utility `/usr/sbin/ss` in the path. If `exe =
-#' "ss"` and this file is found by this function, it will be ignored
+#' "ss"` and this file is found by [check_exe()], it will be ignored
 #' based on the smaller file size relative to the SS3 executable. Linux
-#' users who want to use the workflow of having an SS3 executable in
-#' their path should name the file something else, such as `ss3` or
-#' `ss_linux`. 
-
-#' @author Ian Taylor, Kathryn Doering
+#' users who want to use the workflow of having SS3 in their PATH should
+#' name the SS3 file something besides `ss`, such as `ss3` or
+#' `ss_linux`.
+#' 
+#' @author Ian G. Taylor, Kathryn I. Doering
```

Comment:
Change `I.` to `L.` (BTW, thanks for standardizing names, I'm pretty bad at remembering to consistently use (or not use) a middle initial.